### ISSUE TITLE
How to continute training after 50000 iteration?

### ISSUE THREAD

**Issue by xiaoye-hua** [AlexeyAB/darknet#2633]
Hi.
Due to small learning rate that i set,  the loss is still big after 50000 iteration. But the weights have been saved to final.weights.   
I've tried several times, I can't continue training with the final weights.
Is it possible that I can continue training more iterations ?

**Comment by AlexeyAB** [comment]
@xiaoye-hua Hi,

Set `max_batches=100000` for training up to 100 000 iterations.

**Comment by xiaoye-hua** [comment]
> @xiaoye-hua Hi,
> 
> Set `max_batches=100000` for training up to 100 000 iterations.

Thank you
